src/Blocks/utils.jl

```diff
@@ -42,18 +64,38 @@ Connector with one input signal of type Real.
     ODESystem(Equation[], t, [u...], []; name = name)
 end
 @doc """
-    RealOutput(;name, nout, u_start)
+    RealOutput(;name, u_start)
 
 Connector with one output signal of type Real.
 
 # Parameters:
-- `nout=1`: Number of outputs
-- `u_start=0`: Initial value for `u`
+- `u_start=0`: Initial value for `u`.
 
 # States:
-- `u`: Value of the connector; if nout=1 this is a scalar
+- `u`: Value of the connector which is a scalar.
 """ RealOutput
 
+@connector function RealOutputArray(; name, nout = 2, u_start = zeros(nout))
+    @variables u(t)[1:nout]=u_start [
+        output = true,
+        description = "Inner variable in RealOutput $name"
+    ]
+    u = collect(u)
+    ODESystem(Equation[], t, [u...], []; name = name)
+end
+@doc """
+    RealOutputArray(;name, nout, u_start)
+
+Connector with an array of output signals of type Real.
+
+# Parameters:
+- `nout=2`: Number of outputs.
```

Comment:
That default makes no sense. Why `2`?

Reply:
As this was an array, I thought 2 element might be more common than 1 element for the default. I can change it to 1 if thats better.

Reply:
Why would it have a default at all? This seems like it should be a required argument

Reply:
Yes, that would be better - addressed in [6bb905a](https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/283/commits/6bb905a45acfbdbf2517372ee99697e3c6de236a)